spec/validators/csv_validator_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative './helper_models/csv_validator_helper'
+
+RSpec.describe CsvValidator, type: :model do
+  context 'given a valid file of 10 keywords' do
+    it 'validates to valid' do
+      form = CsvValidatorHelper.new(file_fixture('csv/valid.csv'))
+      expect(form.valid?).to be(true)
+    end
+  end
+
+  context 'given a valid file with blank keywords' do
+    it 'validates to valid' do
+      form = CsvValidatorHelper.new(file_fixture('csv/blank.csv'))
+      expect(form.valid?).to be(true)
+    end
+  end
+
+  context 'given a file with 1001 keywords' do
```

Comment:
If we defined a constant, we can update the context here. The goal is to avoid the magic number, in case it changes, then we no worry about updating the related places like this. 

```suggestion
  context 'given a file with the number of keywords greater than the LIMIT' do
```